pkg/broker/broker.go

```diff
@@ -1073,13 +1074,13 @@ func (a AnsibleBroker) Unbind(
 			return nil, false, jerr
 		}
 
-		if _, err := a.dao.SetState(serviceInstance.ID.String(), apb.JobState{
-			Token:  token,
-			State:  apb.StateInProgress,
-			Method: apb.JobMethodUnbind,
-		}); err != nil {
-			log.Errorf("failed to set initial jobstate for %v, %v", token, err.Error())
-		}
+		// if _, err := a.dao.SetState(bindInstance.ID.String(), apb.JobState{
```

Comment:
Just my opinion but if the code is not running can we delete it? 

We can also go back in history with git and re-add it back?